chirp/drivers/ft450d.py

```diff
@@ -1493,3 +1637,24 @@ def set_settings(self, settings):
                 except Exception as e:
                     LOG.debug(element.get_name())
                     raise
+
+
+class FT450Alias(chirp_common.Alias):
+    """ ALias for Yaesu FT-450 original radio """
+    VENDOR = "Yaesu"
+    MODEL = "FT-450"
+
+
+class FT450ATAlias(chirp_common.Alias):
+    """ ALias for Yaesu FT-450AT radio with tuner """
+    VENDOR = "Yaesu"
+    MODEL = "FT-450AT"

```

Comment:
We're trying not to add aliases anymore, and using subclasses from now on since the clone box handles those properly.

Are you adding these because the communication or feature set is actually different? If not, I think it would be best to not just clutter the download box with choices that are all equivalent. It's one thing if one radio is listed under two completely different models or vendors, but these are almost identical.